src/ParallelExecutor.php

```diff
@@ -266,6 +266,7 @@ private function createProcessLauncher(
         Logger $logger
     ): ProcessLauncher {
         return $this->processLauncherFactory->create(
+            $this->childCommandFactory->phpExecutable,
```

Comment:
to do this we should return the array instead of the string. Currently we are doing:

```php
PhpExecutableFinder::find()
```

which is implemented as:

```php
public static function find(): string
    {
        $phpExecutable = self::getFinder()->find();

        Assert::notFalse(
            $phpExecutable,
            'Could not find the PHP executable.',
        );

        return $phpExecutable;
    }
```

It should rather be:

```php
$executableFinder = new PhpExecutableFinder();
$php = $executableFinder->find(false);
$php = false === $php ? null : array_merge([$php], $executableFinder->findArguments());
```